code/game/machinery/camera/camera.dm

```diff
@@ -76,26 +59,25 @@
 	wires = null
 	cameranet.removeCamera(src) //Will handle removal from the camera network and the chunks, so we don't need to worry about that
 	cameranet.cameras -= src
-	var/list/open_networks = difflist(network,restricted_camera_networks)
-	if(open_networks.len)
-		cameranet.removeCamera(src)
+	cameranet.removeCamera(src)

```

Comment:
Uhhh, this seems like a mistake. I realize it's on TG, as well, but how can you remove a camera you just removed?